client/store/auth/actions.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios'
```

Comment:
Lots of cool stuff here - I am in favor of separating out the actions/thunks/reducers into different files - but if you guys prefer this way that's cool with me.